packages/core/src/components/ProgressBars/GaugeProgress.test.tsx

```diff
@@ -18,6 +18,7 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { wrapInTestApp } from '@backstage/test-utils';
 import { GaugeProgress, getProgressColor } from './GaugeProgress';
```

Comment:
Looks like this is on top of an old version of master btw, needs a fast-forward!